src/index.html

```diff
@@ -13,6 +13,54 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star__block">
+      <div class="star__line stars--0">
+        <div class="star__style star--passive"></div>
+        <div class="star__style star--passive"></div>
+        <div class="star__style star--passive"></div>
+        <div class="star__style star--passive"></div>
+        <div class="star__style star--passive"></div>
+      </div>
+
```

Comment:
Fix classes everywhere
```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>

```

Reply:
<img width="744" alt="image" src="https://github.com/user-attachments/assets/c845af2e-26d4-46fa-bc11-cd958b40333b" />